script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -720,12 +720,12 @@ Leaving aside whether something like that is truly possible or not...
 [sha:3563a49cf5712101f3962638bf549f1839a3ee52]{
 -- Page 717, Offset 26205.
 -- その理屈なら、そいつは人間として生まれて、
-By that logic, that guy is born as a human,
+By that logic, that guy is born as a human.
 }
 [sha:5360d80b2962b15b26360608d04b41076c35e839]{
 -- Page 717, Offset 26206.
 -- 人間として育ち、ある日を境にして、まったく別人の“吸血鬼”になるという事だ。
-gets raised as one, and then one day, a switch is flipped and he becomes someone else entirely―――A 'vampire'.
+He gets raised as one and then, one day, a switch is flipped and he becomes someone else entirely――a 'vampire'.
```

Comment:
hm, not super keen on this one. This pattern reads to me like:
"By that logic, this guy is born as a human. [explanation of how the logic implies that guy is born as a human]" rather than it being a continuation of how he's born, gets raised, then becomes a vampire.

i say revert